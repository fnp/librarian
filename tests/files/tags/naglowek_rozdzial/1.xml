<autor_utworu>Bolesław Prus</autor_utworu>
<dzielo_nadrzedne>Lalka</dzielo_nadrzedne>
<nazwa_utworu>Tom pierwszy</nazwa_utworu>

<naglowek_rozdzial>I. Jak wygląda firma J. Mincel i S. Wokulski przez szkło butelek?</naglowek_rozdzial>

