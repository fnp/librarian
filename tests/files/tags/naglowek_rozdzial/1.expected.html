<div id="book-text">
<div id="toc">
<h2>Spis treści</h2>
<ol>
<a href="#s1" class="anchor">I. Jak wygląda firma J. Mincel i S. Wokulski przez szkło butelek?</a><li>
</ol>
</div>
<h1>
<span class="wl author" id="i1">Bolesław Prus</span><span class="wl collection" id="i2">Lalka</span><span class="wl title" id="i3">Tom pierwszy</span>
</h1>
<h3 class="wl" id="s1">I. Jak wygląda firma J. Mincel i S. Wokulski przez szkło butelek?</h3>
</div>
